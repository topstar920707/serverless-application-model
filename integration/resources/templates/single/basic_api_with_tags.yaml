Resources:
  MyApi:
    Type: AWS::Serverless::Api
    Properties:
      StageName: my-new-stage-name
      DefinitionUri: ${definitionuri}
      Tags:
       TagKey1: TagValue1
       TagKey2: ""
