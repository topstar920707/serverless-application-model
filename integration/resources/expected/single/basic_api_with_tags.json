[
    { "LogicalResourceId":"MyApi", "ResourceType":"AWS::ApiGateway::RestApi" },
    { "LogicalResourceId":"MyApiDeployment", "ResourceType":"AWS::ApiGateway::Deployment" },
    { "LogicalResourceId":"MyApiStage", "ResourceType":"AWS::ApiGateway::Stage" }
]